test/project-config/app_settings/base_settings.json

```diff
@@ -0,0 +1,449 @@
+{
```

Comment:
Can we minimize the settings ? Maybe reducing number of roles , and have something that could work with the setup?

Reply:
Yes I minimize the this one

Reply:
cool